src/badguy/crystallo.hpp

```diff
@@ -24,6 +24,7 @@ class Crystallo final : public WalkingBadguy
 {
 public:
   Crystallo(const ReaderMapping& reader);
+  Crystallo(Vector pos, Vector start_pos, float vel_x, std::unique_ptr<Sprite> sprite, Direction dir, float radius, std::string& script);
```

Comment:
`const Vector&`, `const std::string&`. Parameter list can be split into 2 lines (same for the definition in the `.cpp` file).